rustls/src/tls13/mod.rs

```diff
@@ -198,3 +198,20 @@ impl MessageDecrypter for Tls13MessageDecrypter {
         Ok(msg.into_plain_message())
     }
 }
+
+/// The set of schemes we support for signatures and
+/// that are allowed for TLS1.3.
+///
+/// This prevents (eg) RSA_PKCS1_SHA256 being offered or accepted
+/// in TLS1.3.
+pub(crate) fn is_sigscheme_supported_in_tls13(sigscheme: &SignatureScheme) -> bool {
```

Comment:
small nit: WDYT about dropping the `_in_tls13` suffix. 

At the call-sites, rather than importing the `fn` directly I think it would be clearer to import `tls13` and then call it as `tls13::is_sigscheme_supported`. 

That would remove the redundancy in the fn name but still keeps it clear that the context is TLS 1.3.

Reply:
Yes, I wanted to suggest this too when skimming earlier. Or, go the other way around and just attach this as a method to `SignatureScheme`.

Reply:
have done the latter.